### PR TITLE
docs: Fix ingress example

### PR DESCRIPTION
**Pull request by cneth**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7223

## Description of changes being made

Fix ingress example as suggested by Casey in 

https://mesosphere.slack.com/archives/C01LNF6KML5/p1649278588425029

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4397.s3-website-us-west-2.amazonaws.com/

## Checklist

- [X] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/networking/use-ingress-load-balancer/index.md [hunk 1]
```diff
@@ -48,27 +48,32 @@ Before you begin, you must:
     apiVersion: networking.k8s.io/v1
     kind: Ingress
     metadata:
+      annotations:
+        kubernetes.io/ingress.class: kommander-traefik
+        traefik.ingress.kubernetes.io/router.tls: "true"
+      generation: 7
```
Comment (cbuto) [inline pages/dkp/kommander/2.1/networking/use-ingress-load-balancer/index.md:54]: not a biggie, but we can drop this
```suggestion
```

**Review by cbuto — APPROVED** [review]
🎉

**Comment by mstruebing** [comment]
@cneth manually rebased this PR from `main` to trigger jenkins.
You either need to pull the latest state or force push if you do any changes.

**Review by gvasquezvargas — APPROVED** [review]
One pending comment from Casey. Everything else LGTM!

**Review by gflorinskaya — APPROVED** [review]

**Pull request merged**